collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -547,6 +547,134 @@ def merge(data_dir, subdirs, dataset_name, modes=['train', 'test']):
     return outfold, none_slot
 
 
```

Comment:
for all methods related to dialogflow, can you add dialogflow to the method name?

Reply:
done.